src/components/10-atoms/input-file/story.js

```diff
@@ -1,48 +1,55 @@
 /* global document */
 import { storiesOf } from '@storybook/html';
+import { text, select, boolean, withKnobs } from '@storybook/addon-knobs';
+import { html, render } from 'lit-html';
 import './index';
 import Readme from './README.md';
 
+export const iconOptions = {
```

Comment:
Why is this variable exported? That makes no sense imho.

Reply:
I know now why and we should not do this, because it makes the component a "folder".